.lintstagedrc.mjs

```diff
@@ -23,7 +23,7 @@ export default {
   "*": (files) => {
     const mrlFiles = micromatch(files, [
       "package.json",
-      "**/package.json",
+      // "**/package.json",
```

Comment:
Temporarily disabling this until we fix the issues with dprint and autorelease arguing about whitespace.